docs/REST API/rest-api.md

```diff
@@ -1829,6 +1829,7 @@ the `software` table.
 - `policy_updated_at`: the last time we updated the policy results for the host based on the queries ran.
 - `seen_time`: the last time the host contacted the fleet server, regardless of what operation it was for.
 - `software_updated_at`: the last time software changed for the host in any way.
+- `last_restarted_at`: the last time that the host was restarted
```

Comment:
Just noticed this should have a period at the end to be consistent w/ the rest:
```suggestion
- `last_restarted_at`: the last time that the host was restarted.
```
(I'm gonna go ahead and commit this then approve to avoid delaying this any further)